overlay.nix

```diff
@@ -25,27 +10,27 @@ self: super: {
       sha256 = "1cq36vpsd68g144gn7f3jjkl2bwibqv7nrrrjgvkdj1lfijcwm14";
     };
     version = "5.10.5";
-    kernelPatches = [];
+    kernelPatches = [ ];
     structuredExtraConfig = with super.lib.kernel; {
       CGROUP_FREEZER = yes;
     };
   };
-
   lx2k = self.lib.makeScope self.newScope (self: with self; {
-    rcw = self.callPackage ./rcw { };
+    rcw = self.callPackage ./pkgs/rcw { };
```

Comment:
We need to get the ddrSpeed from somewhere. Not sure if it's better to put that in an extra overlay (as it was in release.nix before) or to somehow pass it into here…